06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,143 @@
+const invertedObject = new InvertedIndex();
+
+
+const correctBook = require('./book.json');
+const wrongBook = require('./wrongFormat.json');
+const zeroLength = require('./length.json');
+const book = require('./News');
+const emptyFile = require('./empty');
+const rightFile = require('./rightBook');
+
```

Comment:
A base filename should exactly match the name of its default export. Please refer to your [Airbnb Style guide](https://github.com/airbnb/javascript#naming--filename-matches-export)